Tests/Unit/ExtensionTest.php

```diff
@@ -79,14 +78,21 @@ private function getValidConfiguration(array $options): array
                 'resource_server' => [
                     'public_key' => 'foo',
                 ],
-                'persistence' => [],
+                //Pick one for valid config:
+                //'persistence' => ['doctrine' => []]
+                'persistence' => ['in_memory' => 1],
             ],
         ];
     }
 
     private function setupContainer(ContainerBuilder $container): void
     {
         $container->register(ScopeManager::class);
-        $container->setAlias(ScopeManagerInterface::class, ScopeManager::class);
+        //the line below is no-longer needed if "persistence" above
+        //is set to either doctrine or in_memory
+        //which will ensure that services defined in either
+        // storage/in_memory.xml or storage/doctrine.xml
+        //get properly defined
+        //$container->setAlias(ScopeManagerInterface::class, ScopeManager::class);
```

Comment:
This change seems unrelated to Symfony 5.

Reply:
I would recommend keeping this. See my comment on the previous issue.

Reply:
@elchris Sure, just please delete all these commented lines.

Reply:
ok, done, pushed!

Reply:
:+1: